### ISSUE TITLE
Release 8.0.0

### ISSUE THREAD

**Issue by ashfurrow** [Moya/Moya#759]
We were waiting on ReactiveCocoa/ReactiveSwift to be finalized for Swift 3 before we release. They've released a stable version for Swift 3, so we should consider turning our latest beta into a final release. We haven't seen many breaking changes since the last beta, so I don't anticipate this will be a problem, but I thought I'd open an issue to discuss.

**Comment by AndrewSB** [comment]
`ReactiveCocoa/ReactiveSwift` looks like it's still an alpha - are you saying that their alpha release is stable on Swift 3, and would make sense to vend a ReacticeSwift alpha in a stable version of Moya?

**Comment by ashfurrow** [comment]
Ah, sorry, you're right. Got it confused with [ReactiveObjC](https://github.com/ReactiveCocoa/ReactiveObjC). Let's hold off then.

**Comment by AndrewSB** [comment]
We could either close this, or add that as a milestone to track the release - whatever you prefer ash

**Comment by ashfurrow** [comment]
Yeah, could you create a milestone? It would be awesome to keep this stuff documented 🎉

**Comment by AndrewSB** [comment]
Closing this in favor of the milestone (https://github.com/Moya/Moya/milestone/6)

**Comment by BasThomas** [comment]
`ReactiveSwift` seems to be really close to their `1.0` release, as they have pushed their first release candidate! 🎉

Is there anything we should tackle ourselves for `8.0.0` still? @Moya/contributors

**Comment by ashfurrow** [comment]
Not that I'm aware of.
